portfolio/src/main/webapp/script.js

```diff
@@ -45,26 +45,62 @@ function openConnectModal() {
   modal.style.display = "block";
 }
 
-function closeIntroModal(){
+function openQuestionModal() {
+  const modal = document.getElementById("myQuestionModal");
+  modal.style.display = "block";
+}
+
+function openViewModal() {
+  const modal = document.getElementById("myQuestionViewModal");
+  modal.style.display = "block";
+  fetch('/data').then(response => response.json()).then((Questions) => {
+  const questionContainer = document.getElementById("question-container");
+  questionContainer.innerHTML = '';
+  var i;
+  for (i = 0; i < Questions.length; i++) {
+    var createPelement = document.createElement("LI");
+    var createNode = document.createTextNode(Questions[i].name + " asked: " + Questions[i].text);
+    createPelement.appendChild(createNode);
+    questionContainer.appendChild(createPelement);
+    if (i === 9) break;
```

Comment:
why break at 9? move to a class level const?

Reply:
Because that is the max number I want the user to see on their screen for viewing purposes. Also what do you mean "move to a class level const"?